src/main/kotlin/uk/gov/justice/digital/hmpps/sendlegalmailtoprisonsapi/validators/RequestBodyValidators.kt

```diff
@@ -0,0 +1,18 @@
+package uk.gov.justice.digital.hmpps.sendlegalmailtoprisonsapi.validators
+
+import uk.gov.justice.digital.hmpps.sendlegalmailtoprisonsapi.barcode.CreateBarcodeRequest
+import uk.gov.justice.digital.hmpps.sendlegalmailtoprisonsapi.config.MalformedRequest
+import uk.gov.justice.digital.hmpps.sendlegalmailtoprisonsapi.config.ValidationException
+import uk.gov.justice.digital.hmpps.sendlegalmailtoprisonsapi.contact.CreateContactRequest
+
+fun validateRequestHasDobOrPrisonNumber(requestBody: CreateContactRequest) {
```

Comment:
No class - marvellous! However I think we are missing some unit tests for these functions?

Reply:
Yeah, you're probably right 🤷‍♂️ 
(I took the view that they were so simple and were being implicitly tested via the integrations tests, but yeah, I'll knock up a couple of unit tests 👍 )